File: app/src/main/java/hungry/ex_frag/numSample/Item.java
package hungry.ex_frag.numSample;

import java.io.Serializable;

public class Item implements Serializable {
    private String name;
    private String cha;
    private String des;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCha() {
        return cha;
    }

    public void setCha(String cha) {
        this.cha = cha;
    }

    public String getDes() {
        return des;
    }

    public void setDes(String des) {
        this.des = des;
    }
}

File: flutter_app/lib/models/item.dart
/// Item - 숫자-인물 변환 시스템 데이터 모델
/// 00-99까지의 숫자를 인물로 변환하는 PAO 시스템
class Item {
  final String name; // 인물 이름
  final String cha; // 특징/캐릭터
  final String des; // 설명

  Item({
    required this.name,
    required this.cha,
    required this.des,
  });
}
